CSC322-Grinnell/mayflower-directory
# Clear files

Resolves #186, resolves #171 

## Changes
* Remove 'Change Password' button from Log in view. 
* Delete duplicated views in User (i.e. Session, Unlock, etc.) + route to views under Devise.  
* Delete PasswordReset controller + views <= do not work. Password reset is handled in User controller. 
* Make profile strong params private 
* Delete Log in view that contains Sign up button. 
* Add ```:admin``` attribute to ```configure_sign_up_params```. This allows new registration to be assigned with admin privileges. The model already restricts non-admin users to create new accounts. 
* Remove various files relating to Mailer. Our Mailer does not work. 

## Testing: 
I mostly deleted the views under User. To verify my changes are correct: 
* Test Log In, Create New User, Change Password and Log Out are working correctly.
* Create New User should allow creating a new admin account. 
* Log in view should have no "Sign up" button. 
* Test that Picture is working correctly.
